docs/root/configuration/http_filters/router_filter.rst

```diff
@@ -27,17 +27,16 @@ ingress/response path. They are documented in this section.
 x-envoy-max-retries
 ^^^^^^^^^^^^^^^^^^^
 
-If a :ref:`retry policy <config_http_conn_man_route_table_route_retry>` is in place, Envoy will default to retrying one
-time unless explicitly specified. The number of retries can be explicitly set in the
-:ref:`route retry config <config_http_conn_man_route_table_route_retry>`  or by using this header.
-If a :ref:`retry policy <config_http_conn_man_route_table_route_retry>` is not configured and
-:ref:`config_http_filters_router_x-envoy-retry-on` or
-:ref:`config_http_filters_router_x-envoy-retry-grpc-on` headers are not specified, Envoy will not retry a failed request.
+If a :ref:`retry policy <envoy_api_field_route.RouteAction.retry_policy>` is in place, Envoy will default to retrying
+one time unless explicitly specified. The number of retries can be explicitly set in the route retry config or by using
+this header. If a retry policy is not configured and :ref:`config_http_filters_router_x-envoy-retry-on` or
```

Comment:
These two are still v1 API refs.

Reply:
I thought this header document is not v1 API one. https://94844-65214191-gh.circle-artifacts.com/0/source/generated/docs/configuration/http_filters/router_filter.html#config-http-filters-router-x-envoy-retry-on. Is that correct?

The link is in https://94844-65214191-gh.circle-artifacts.com/0/source/generated/docs/configuration/http_filters/router_filter.html#x-envoy-max-retries.

Reply:
Yep, you're right here, please ignore my comment.